uninstall.sh

```diff
@@ -226,6 +249,11 @@ git config --global --unset githooks.previous.searchdir
 git config --global --unset githooks.disable
 git config --global --unset alias.hooks
 
+if using_hooks_path; then
+    git config --global --unset githooks.use.hookspath
+    git config --global --unset core.hooksPath
```

Comment:
Maybe we should set the `githooks.use.core.hooksPath` variable to the same as `core.hooksPath` so here we could check if they match and only unset if they do. This way we could make sure we don't unset preexisting settings from the user.